packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -89,6 +89,7 @@ export const messages = {
   force_source_deploy_select_manifest:
     'You can run SFDX: Deploy Source in Manifest to Org only on a manifest file.',
   force_source_retrieve_text: 'SFDX: Retrieve Source from Org',
+  force_source_retrieve_display_text: 'Retrieve Source from Org',
```

Comment:
@hbc-bindu This is the text that would display when hovering over the 'Retrieve' icon on individual components.